docs/modules/reference/pages/telemetryd/protocols/jti.adoc

```diff
@@ -116,3 +116,47 @@ The following globals will be passed to the script:
 | Decoded JTI message from which the metrics should be extracted.
 | org.opennms.netmgt.telemetry.adapters.jti.proto.TelemetryTop
 |===
+
+NOTE: In OpenNMS, logs are essential for monitoring and troubleshooting. By default, the log message size is limited to 500 characters. However, in some cases, such as when viewing detailed proto object logs, you may need to increase the character size to capture the full log details.
+
+=== Steps to Increase Log Character Size
+Locate file `org.ops4j.pax.logging.cfg`  in the `${OPENNMS_HOME}/etc/` directory.
```

Comment:
Insert following statement after ===Steps to Increase   and remove above note.

By default, the log message size is limited to 500 characters with karaf logs. However, in some cases, such as when viewing detailed proto object logs, you may need to increase the character size to capture the full log details.